add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code directly uses raw SQL queries with user input, which is vulnerable to SQL injection attacks. Additionally, the code is procedural and lacks separation of concerns.

Recommendations:
- Use prepared statements to prevent SQL injection. Here's an example using MySQLi:
  ```php
  $stmt = $connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
  $stmt->bind_param("ssi", $title, $author, $year);
  $stmt->execute();
  $stmt->close();
  ```
- Consider refactoring the code to use an object-oriented approach. Create a `Book` class and a `BookRepository` class to handle database operations. This will improve maintainability and testability.
- Example of a `Book` class:
  ```php
  class Book {
      private $title;
      private $author;
      private $year;

      public function __construct($title, $author, $year) {
          $this->title = $title;
          $this->author = $author;
          $this->year = $year;
      }

      // Getters and setters...
  }
  ```
- Example of a `BookRepository` class:
  ```php
  class BookRepository {
      private $connection;

      public function __construct($connection) {
          $this->connection = $connection;
      }

      public function addBook(Book $book) {
          $stmt = $this->connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
          $stmt->bind_param("ssi", $book->getTitle(), $book->getAuthor(), $book->getYear());
          $stmt->execute();
          $stmt->close();
      }

      // Other CRUD methods...
  }
  ```